### ISSUE TITLE
Max Buffer Issue

### ISSUE THREAD

**Issue by mknayak** [mckn/gulp-nuget#19]
I am looping through lots of proejcts and generating nuget packages. its throwing halfway statng Process ```
terminated with code 1.
				if(err) throw new gutil.PluginError('gulp-nuget', err);
				        ^
Error: stdout maxBuffer exceeded
    at Socket.<anonymous> (child_process.js:266:14)
```
Can you please add support to provide max-buffer parameter.

**Comment by mckn** [comment]
Thanks! Will fix this asap.

**Comment by mckn** [comment]
This is resolved in the latest version 1.2.0
